### DISCUSSION TITLE
Custom Videos

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [ThioJoe/YT-Spammer-Purge#603] [Ideas]
Right now, I believe there are 2 options to delete comments:
1: Delete with one video
2: Delete comments by last few videos from channel (you get to decide, how many last videos, ex: 2)

But, a way to select one video, alternate video doesn't exist.

For eg: 
Video A and Video c

You can kinda do this using the delete by last few videos on channel, but then the comments will get deleted out of video B too unless you exclude them.

**Comment by ThioJoe** [comment]
You can scan a list of any videos using the "Scan specific video**s**" mode already. Unless you mean something else?

**Reply by KendallDoesCoding** [reply]
> You can scan a list of any videos using the "Scan specific video**s**" mode already. Unless you mean something else?

Ok, I haven't seen it that's why I asked, will have a look in a bit. Thanks
